gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -211,8 +211,8 @@ class NFPNodeDriver(driver_base.NodeDriverBase):
         pconst.LOADBALANCER, pconst.FIREWALL, pconst.VPN]
     SUPPORTED_SERVICE_VENDOR_MAPPING = {
         pconst.LOADBALANCER: ["haproxy"],
-        pconst.FIREWALL: ["vyos", "nfp"],
-        pconst.VPN: ["vyos"],
+        pconst.FIREWALL: ["vyos", "nfp", "asav"],
+        pconst.VPN: ["vyos", "asav"],
```

Comment:
This list should not be filled like this, as we don't submit asav. This needs to be filled through a config file with mappings of service type to supported vendors.